dashboard/test/controllers/pd/workshop_certificate_controller_test.rb

```diff
@@ -129,6 +129,9 @@ def expect_renders_text(mock_background_image, string)
     # :trim is called in order to make sure excess empty space is removed around
     # the text.
     mock_text_image.expects(:trim!).returns(mock_text_image)
+    # :columns is the width of the image. This is checked to see if the text needs
+    # to be resized.
+    mock_text_image.expects(:columns).returns(325)
```

Comment:
Is this is px or something else?  325px seems small unless I'm missing something.

Reply:
Yes it is pixels